force-app/main/default/objects/Customizable_Rollup_Settings__c/fields/Limit_Recalculated_Recurring_Donations__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Limit_Recalculated_Recurring_Donations__c</fullName>
```

Comment:
We've been using CamelCase for most new fields vs. Snake_Case. Should the three fields follow the CamelCase format?